GUIDELINES.adoc

```diff
@@ -134,3 +134,11 @@ Use the following template when adding a new style guide entry:
 * Avoid linking to other resources to provide the style guidance. Incorporate the guidance from other resources into the entry description.
 
 * Use single lines with no breaks for each paragraph. In other words, do not hard wrap at 80 characters, for example.
+
+* Ensure that you use italics when introducing or defining a term. The following examples demonstrate the use of this guideline:
```

Comment:
Small suggestion for minimalism and prescriptiveness you could remove the 'Ensure that you'

Reply:
Thanks, @ndeevy . Good suggestion. I removed the opening clause from both bullet points.